observing_examples.ipynb

```diff
@@ -38,16 +41,54 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "First we are going to download a model spectrum for HAT-P-11 from <a href=\"http://phoenix.astro.physik.uni-goettingen.de/\">PHOENIX</a>:"
+    "First we get the star's parameters using <a href=\"https://astroquery.readthedocs.io/en/latest/\">astroquery's</a> Gaia query: "
    ]
   },
   {
    "cell_type": "code",
-   "execution_count": 43,
+   "execution_count": 114,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Query finished.\n"
+     ]
+    }
+   ],
+   "source": [
+    "ID = 'HAT-P-11'\n",
+    "\n",
+    "coord = SkyCoord.from_name(ID)\n",
+    "width = u.Quantity(1, u.arcmin)  # width / height of search\n",
+    "height = u.Quantity(1, u.arcmin)\n",
+    "search_results = Gaia.query_object_async(coordinate=coord, width=width, height=height)\n",
+    "\n",
+    "# the queried star will be the one nearest to the given coordinates\n",
+    "search_results.add_index('dist', unique=True)\n",
+    "hatp11_info = search_results.loc['dist', min(search_results['dist'])]\n",
+    "\n",
+    "# now get the parameters\n",
+    "stellar_radius = hatp11_info['radius_val'] * u.R_sun\n",
+    "distance = (1 / (hatp11_info['parallax']) * 1000) * u.pc\n",
+    "T_eff = hatp11_info['teff_val'] * u.K"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Next we are going to download a model spectrum for HAT-P-11 from <a href=\"http://phoenix.astro.physik.uni-goettingen.de/\">PHOENIX</a>:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 115,
    "metadata": {},
    "outputs": [],
    "source": [
-    "T_eff = 4800 # approx temperature of hatp11[K], Bakos et al. 2010\n",
+    "T_eff = round(T_eff.value/100) * 100  # round to nearest 100 K\n",
```

Comment:
You can actually round integers with the `round` function if you supply it with a second argument that's a negative number, like this: 
```
>>> round(4781, -2)
4800
```
which is just a tad cleaner.